openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -263,12 +263,8 @@ jQuery(function () {
 
     $('#wikiselect').on('focus', function(){$(this).trigger('select');})
 
-    // Clicking outside of menus closes menus
-    $(document).on('click', function (event) {
-        const $openMenus = $('.checkbox-menu :checked').parents('.checkbox-menu');
-        $openMenus
-            .filter((_, menu) => !$(event.target).closest(menu).length)
-            .find('[type=checkbox]')
-            .prop('checked', false);
+    // Open one dropdown at a time.
+    $('.header-dropdown details').on('click', function () {
```

Comment:
This doesn't work the same as it doesn't catch clicks outside the menu. Please use ` $(document).on('click', function (event) {` like before.

Reply:
Done.